src/devices/video/hd61830.cpp

```diff
@@ -59,14 +59,15 @@ static constexpr int CYCLES[] =
 	4, 4, 4, 4, 4, -1, -1, -1, 4, 4, 4, 4, 6, 6, 36, 36
 };
 
-static constexpr int MODE_EXTERNAL_CG      = 0x01;
-static constexpr int MODE_GRAPHIC          = 0x02;
-static constexpr int MODE_CURSOR           = 0x04;
-static constexpr int MODE_BLINK            = 0x08;
-static constexpr int MODE_MASTER           = 0x10;
-static constexpr int MODE_DISPLAY_ON       = 0x20;
-
-
+enum
+{
+	MODE_EXTERNAL_CG = 0x01,
+	MODE_GRAPHIC     = 0x02,
+	MODE_CURSOR      = 0x04,
+	MODE_BLINK       = 0x08,
+	MODE_MASTER      = 0x10,
+	MODE_DISPLAY_ON  = 0x20,
+};
```

Comment:
This doesn’t really gain anything – you’ve still got these symbols at global scope within the file.

Reply:
Yeah, as the commit message said, it's just slightly clearer that these are all related (as an `enum`).
If minor refactorings are not accepted, I can take this commit out of the stack.

Reply:
It’s still an anonymous enum, it still has `int` storage class, the enumeration constants end up in the global namespace for the file.  You can infer that they’re related because they have a `MODE_` prefix and are declared together.  It really doesn’t gain anything.

Reply:
Are you criticising those flaws because you think if those were fixed, this would actually be better, or just for criticism's sake? Unclear what should be done here.

Reply:
Well, you could potentially turn it into an enum nested inside the class in the header with an explicit storage class, and declare bitwise operators for it so that logical operations return the same enum type (there’s a helper macro for that part, and a macro for allowing an enum to be used in save states).  That gives you a little safety when dealing with it.  However, just putting the constants in an anonymous enum here feels like churn for the sake of it.  The question is whether it’s worth the trouble or not.  I’d just leave it as-is.